monolith/src/main/java/com/society/server/web/CommentController.java

```diff
@@ -16,31 +17,36 @@
 public class CommentController {
 
     private final CommentService commentService;
+    private final PostService postService;
 
-    public CommentController(CommentService commentService) {
+    public CommentController(CommentService commentService, PostService postService) {
         this.commentService = commentService;
+        this.postService = postService;
     }
 
-    @PostMapping()
+
+    @PostMapping("/comment")
     public ResponseEntity<CommentDTO> createComment(@RequestHeader("X-username") String username,
+                                                    @RequestParam("postId") Long id,
                                                     @Valid @RequestBody CreateCommentDTO createCommentDTO) {
-        CommentDTO commentDTO = commentService.createComment(username, createCommentDTO);
-
+        CommentDTO commentDTO = commentService.createComment(id, username, createCommentDTO);
         return ResponseEntity
                 .status(HttpStatus.CREATED)
                 .body(commentDTO);
     }
 
-    @DeleteMapping("/commentId")
-    public ResponseEntity<CommentDTO> deleteComment(@PathVariable("commentId") Long id) {
+    //Todo: Fix deleteComment method
+    @DeleteMapping("/{commentId}")
```

Comment:
I think the path needs to be `/comment/{commentId}` like the createComment to not mess up with the path vars

Reply:
According to convention it should be "/comments/{commentId}". Тhe action to be taken depends on the type of HTTP method. 
Anyways, this method is not ready, it will be fixed in next Pull request.